easybuild/easyconfigs/f/FLASH2/FLASH2-20160507-goolf-1.7.2064826f4.eb

```diff
@@ -0,0 +1,38 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'MakeCp'
+
+name = 'FLASH2'
+version = '20160507'
+versionsuffix = '64826f4' # git commit to download
```

Comment:
@pescobar what's the point in exposing this?

why not:

```python
commit = '64826f4'
...
sources = ['%s.tar.gz' % commit]
```

Of course, the real 'fix' would be to support checking out Git repositories as a download mechanism...

Reply:
We have an alternate approach for this now, i.e. to rename the source tarball on download, see http://easybuild.readthedocs.io/en/latest/Writing_easyconfig_files.html#alternative-formats-for-sources